sdk/python/packages/flet-core/src/flet_core/control.py

```diff
@@ -481,6 +482,9 @@ def _build_command(self, update=False):
                 sval = str(val).lower()
             elif isinstance(val, dt.datetime) or isinstance(val, dt.date):
                 sval = val.isoformat()
+            elif dataclasses.is_dataclass(val):
+                sval = dataclasses.asdict(val)
+                sval = json.dumps(sval)
             else:
```

Comment:
I like the idea of serializing dataclass attributes centrally - thanks for the idea!

But at the moment we are doing that in `_before_build_command()` method, like for example here: https://github.com/flet-dev/flet/blob/main/sdk/python/packages/flet-core/src/flet_core/text_span.py#L45-L47

I'd stick to this method for now, until we decide to migrate to your method - I don't know how it's going to affect existing functionality 🧐

Reply:
@FeodorFitsner Thank you for your feedback. It's nice to be helpful for such a cool project. Have you considered pydantic as a serializer? I could help with rewriting the code using pydantic.

Reply:
Absolutely, I'd love to look into improving the thing with Pydantic - let's kick off the discussion after New Year ;)
For your PR we could still apply old-school approach :)

Reply:
Could you change to `_before_build_command()` please?

Reply:
> Could you change to `_before_build_command()` please?

complete